x-pack/platform/plugins/shared/inference/server/chat_complete/callback_api.ts

```diff
@@ -84,7 +84,7 @@ export function createChatCompleteCallbackApi({
       abortSignal,
       stream,
       maxRetries = 3,
-      retryConfiguration = { retryOn: 'all' },
+      retryConfiguration = {},
```

Comment:
What's the reason behind this change?

Reply:
It should just use the defaults from retryConfiguration (which is: only retryable errors like 503 etc). I will backport this separately too.